### PR TITLE
program to detect the middle element of a linkedlist in c is added

### PR DESCRIPTION
**Pull request by sakshi300699**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

C program to detect the middle element of the linkedlist using two-pointer approach.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Issue Addressed- #1617

### CODE REVIEW

File: C/ds/middle_element_of_linkedlist.c [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  
+/* Driver function*/
+int main()  
+{  
+    
+    struct Node* head = NULL;  
+    int i;
+    printf("Enter the number of elements in the linkedlist "); 
+    scanf("%d",&i); 
+
```
Comment (srimanikantapalakollu) [inline C/ds/middle_element_of_linkedlist.c]: @sakshi300699 please remove these extra lines

File: C/ds/middle_element_of_linkedlist.c [hunk 2]
```diff
@@ -0,0 +1,92 @@
+/*
```
Comment (srimanikantapalakollu) [inline C/ds/middle_element_of_linkedlist.c:1]: @sakshi300699 Remove all the extra unnecessary spaces in the code.

File: C/ds/middle_element_of_linkedlist.c [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+Enter the element 3
+Enter the element 1
+Enter the element 6
+Enter the element 7
+The middle element is 1
+
+
+
+*/
+
```
Comment (srimanikantapalakollu) [inline C/ds/middle_element_of_linkedlist.c]: @sakshi300699 Remove this extra lines from the end of the file.

**Review by srimanikantapalakollu — CHANGES_REQUESTED** [review]

**Comment by sakshi300699** [comment]
@srimani-programmer Done the required changes. Please review and also kindly add the necessary level tag.

**Comment by sakshi300699** [comment]
Please review my PR, project maintainers

**Comment by sakshi300699** [comment]
It's been 16 days since I made this PR. Please review the PR @HarshCasper  @srimani-programmer

**Review by HarshCasper — APPROVED** [review]

**Comment by sakshi300699** [comment]
@HarshCasper @srimani-programmer Please add DWOC tag to this

**Comment by sakshi300699** [comment]
@HarshCasper @atarax665 @anushkrishnav @srimani-programmer please add DWOC tag to this PR.

**Pull request merged**